product_lot_sequence/__manifest__.py

```diff
@@ -7,8 +8,9 @@
         Adds ability to define a lot sequence from the product""",
     "version": "14.0.1.0.2",
     "license": "AGPL-3",
-    "author": "ForgeFlow S.L., Odoo Community Association (OCA)",
+    "author": "Ooops, cetmix, ForgeFlow S.L., Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/product-attribute",
+    "maintainers": ["CetmixGitDrone"],
```

Comment:
**issue**: Using company-wide accounts was discussed in a previous PR https://github.com/OCA/stock-logistics-warehouse/pull/1890 , from what I understand it is not accepted.

**suggestion**: Add yourself and optionally other people from your company, instead.

Reply:
> **issue**: Using company-wide accounts was discussed in a previous PR [OCA/stock-logistics-warehouse#1890](https://github.com/OCA/stock-logistics-warehouse/pull/1890) , from what I understand it is not accepted.
> 
> **suggestion**: Add yourself and optionally other people from your company, instead.
 Hello @aleuffre thank you for mentioning it. Although it was a subject of a previous discussion there is nothing in the official OCA documents prohibiting it. I would appreciate your pointing at this document in case I missed something. Another option is to remove this part from the manifest completely or put someone else who would like to take care of maintaining this module.